### DISCUSSION TITLE
confusion about patch in help.html

### DISCUSSION THREAD

**Discussion by hycinth22** [pedrolcl/VMPK#10] [Q&A] [answered]
I am translating help.html and confused about this line.

https://github.com/pedrolcl/VMPK/blob/3e2616bc85c63e71957c149e3af92e4b27c9a296/data/help.html#L182-L183

could you explain that what is patch name? 

and this one
https://github.com/pedrolcl/VMPK/blob/3e2616bc85c63e71957c149e3af92e4b27c9a296/data/help.html#L179-L180

what is ` patches with a limited range of notes`.

---

is patch a typo for pitch? or means something else?

@pedrolcl  
thx

**Comment by pedrolcl** [comment, marked as answer]
"patch" in this context is equivalent to "instrument" or "sound". In General MIDI, there is a list of 128 predefined patches, grouped by instrument families. Patch  #1 is Grand Piano, patch #41 is Violin,  #57 is Trumpet., and so on.

See: https://www.midi.org/specifications-old/item/gm-level-1-sound-set

This are only names, not strict definitions, so the same patch may sound very different on several synthesizers.

The _tesitura_ of an instrument or patch is the range of notes supported by the instrument. On real instruments, there is usually a lower note limit, for instance Violins have a lower note of G3, but some synths may or may not respect that limit.

**Reply by hycinth22** [reply]
thanks
